examples/mnist-pytorch/client/fedn.yaml

```diff
@@ -2,4 +2,5 @@ entry_points:
   train:
     command: /venv/bin/python entrypoint train $ENTRYPOINT_OPTS
   validate:
-    command: /venv/bin/python entrypoint validate $ENTRYPOINT_OPTS
\ No newline at end of file
+    command: /venv/bin/python entrypoint validate $ENTRYPOINT_OPTS
+helper: pytorchhelper
```

Comment:
what is this?

Reply:
Ah, it sneakt in. I was thinking about a new way to specify the helper. Will remove for now.